### ISSUE TITLE
Feature: Automatic releases

### ISSUE THREAD

**Issue by nonara** [LeDDGroup/typescript-transform-paths#128]
## Edit

Proposal is to automatically create a GH release based on `standard-version` changelog. I tried with one action package that was written in Ruby, but it fails.

I intend to write a new GH action at some point, and when/if I do, I will implement into this package

**Comment by danielpza** [comment]
Hi @nonara sorry for the late response, I'll take a look asap.

**Comment by danielpza** [comment]
Hello @nonara, I'm not really sure what's needed, I was taking a look at the docs and https://github.blog/changelog/2021-04-20-github-actions-control-permissions-for-github_token/ states:

> The GITHUB_TOKEN is an automatically generated secret that lets you make authenticated calls to the GitHub API in your workflow runs.

so the `GITHUB_TOKEN` variable should already be defined for the actions automatically?
Please point me to the relevant documentation if that's not the case so I can learn more about it.

**Comment by nonara** [comment]
@danielpza Ah, interesting! I'm fairly positive that you used to need to generate one and store it in the repo secrets. It's been awhile since I've set one up, but I believe the last time I did pre-dates that changelog. Maybe the auto-generation is a new feature... Or maybe my memory is just slipping 😂 Anyway, thanks for checking into it! 

Unfortunately, the action I found seems to fail. The logic doesn't seem to be very sound either, so I think I'm going to create a new GH action repo that parses the changelog into markdown AST. That should be a little more reliable to ensure we're getting the right details.

Anyway, good to hear from you again. I hope all is going well for you!
